catalog/format/iceberg/src/main/java/org/projectnessie/catalog/formats/iceberg/nessie/IcebergConstants.java

```diff
@@ -76,5 +85,11 @@ private IcebergConstants() {}
           CURRENT_SNAPSHOT_TIMESTAMP,
           CURRENT_SCHEMA,
           DEFAULT_PARTITION_SPEC,
-          DEFAULT_SORT_ORDER);
+          DEFAULT_SORT_ORDER,
+          NESSIE_CONTENT_ID,
+          NESSIE_COMMIT_ID,
+          NESSIE_COMMIT_REF);
+
+  public static final Set<String> TRANSIENT_PROPERTIES =
+      ImmutableSet.of(NESSIE_CONTENT_ID, NESSIE_COMMIT_ID, NESSIE_COMMIT_REF);
```

Comment:
Not sure about the "transient" term here... Content and Commit IDs never change or go away :thinking: How about `INTERNAL_PROPERTIES`?

Reply:
Transient in the sense that these properties are removed/ignored when received but added (so not stored)

Reply:
Still "transient" looks confusing to me in this context. How about "derived" properties?